src/server/pps/server/api_server.go

```diff
@@ -2232,6 +2255,8 @@ type jobOptions struct {
 	volumes            []api.Volume
 	volumeMounts       []api.VolumeMount
 	imagePullSecrets   []api.LocalObjectReference
+	cpuFootprint       float32
+	memFootprint       string
```

Comment:
Cool. Yea we want this to be shown for inspect pipeline and inspect job. So I think the pachctl output commands need to be updated to print this information as well.